test/plugins/link.js

```diff
@@ -26,7 +29,7 @@ describe('Link plugin', function() {
     });
 
     this.irc.createMessage({
-      message: 'http://localhost:9002/basic'
+      message: 'http://127.0.0.1:9002/basic'
```

Comment:
is this actually fixing things ? if not, please revert this change, cause it enforces IPv4 over IPv6.